packages/ifpack2/src/Ifpack2_Details_Chebyshev_def.hpp

```diff
@@ -481,6 +485,10 @@ setParameters (Teuchos::ParameterList& plist)
     // userInvDiag.
   }
 
+  // Load the kernel fuse override from the parameter list
+  if (plist.isParameter ("chebyshev: use native spmv"))
+    ckUseNativeSpMV = plist.get("chebyshev: use native spmv", ckUseNativeSpMV);
```

Comment:
This should also be added to the list of allowable Ifpack2 options, in `Ifpack2_Parameters.cpp`.

Reply:
Will do.

Reply:
Fixed.